lib/screens_web/views/audio_view.ex

```diff
@@ -5,8 +5,7 @@ defmodule ScreensWeb.AudioView do
 
   @spec render_pill_header(atom(), String.t() | nil) :: Phoenix.HTML.safe()
   defp render_pill_header(pill, wayfinding) do
-    ~E|<s><%= render_pill(pill) %> trips<%= render_wayfinding(wayfinding) %></s>
-|
+    ~E|<s><%= render_pill(pill) %> trips<%= render_wayfinding(wayfinding) %></s>|
```

Comment:
I think some of these weird line breaks were there to make the SSML output more readable while debugging, but at this point we know it all works so it's fine to remove them.